kitsune/upload/storage.py

```diff
@@ -6,10 +6,19 @@
 from django.conf import settings
 from django.core.files.storage import FileSystemStorage
 
+from storages.backends.gcloud import GoogleCloudStorage
 from storages.backends.s3boto3 import S3Boto3Storage
 
+DjangoStorage = None
```

Comment:
Nit. Since the `else` clause takes care of the default case, we could drop this if you'd like.

Reply:
thanks @escattone! removed that line :+1: also, removed `GS_LOCATION`, it's not required for the GCS backend to work :slightly_smiling_face: